tls/s2n_psk.c

```diff
@@ -163,3 +217,87 @@ int s2n_psk_verify_binder(struct s2n_connection *conn, struct s2n_psk *psk,
 
     return S2N_SUCCESS;
 }
+
+static S2N_RESULT s2n_psk_write_binder(struct s2n_connection *conn, struct s2n_psk *psk,
+        const struct s2n_blob *binder_hash, struct s2n_stuffer *out)
+{
+    ENSURE_REF(binder_hash);
```

Comment:
I suggest
```c
GUARD_AS_RESULT(s2n_blob_is_valid(binder_hash));
```

Reply:
Don't we need a valid blob for `binder_hash`, so we can be sure to use it to initialize `binder` and for `s2n_psk_calculate_binder`?